peer/brontide_test.go

```diff
@@ -1118,3 +1118,136 @@ func TestPeerCustomMessage(t *testing.T) {
 	require.Equal(t, remoteKey, receivedCustom.peer)
 	require.Equal(t, receivedCustomMsg, &receivedCustom.msg)
 }
+
+func TestPeerUnknownMessage(t *testing.T) {
+	testCases := []struct {
+		name          string
+		messageType   lnwire.MessageType
+		messageData   []byte
+		shouldClose   bool
+	}{
+		{
+			name:        "unknown message odd",
+			messageType: lnwire.MessageType(32001),
+			messageData: []byte{4, 5, 6},
+			shouldClose: false,
+		},
+		{
+			name:        "unknown message even",
+			messageType: lnwire.MessageType(32000),
+			messageData: []byte{4, 5, 6},
+			shouldClose: true,
+		},
```

Comment:
Perhaps we could add a case where `messageType > CustomTypeStart` and is even to ensure we still disconnect for custom even messages.